contracts/ChildChainGaugeInjectoooor.sol

```diff
@@ -190,40 +197,49 @@ contract ChildChainGaugeInjector is ConfirmedOwner, Pausable, KeeperCompatibleIn
      * @notice Injects funds into the gauges provided
    * @param ready the list of gauges to fund (addresses must be pre-approved)
    */
-    function injectFunds(address[] memory ready) public whenNotPaused {
+    function _injectFunds(address[] memory ready) internal whenNotPaused {
         uint256 minWaitPeriodSeconds = s_minWaitPeriodSeconds;
         address tokenAddress = s_injectTokenAddress;
         IERC20 token = IERC20(tokenAddress);
-        address[] memory gaugeList = s_gaugeList;
         uint256 balance = token.balanceOf(address(this));
         Target memory target;
 
         for (uint256 idx = 0; idx < ready.length; idx++) {
             target = s_targets[ready[idx]];
-            IChildChainGauge gauge = IChildChainGauge(gaugeList[idx]);
+            IChildChainGauge gauge = IChildChainGauge(ready[idx]);
             uint256 period_finish = gauge.reward_data(tokenAddress).period_finish;
 
             if (
                 target.lastInjectionTimeStamp + s_minWaitPeriodSeconds <= block.timestamp &&
                 period_finish <= block.timestamp &&
                 balance >= target.amountPerPeriod &&
-                target.periodNumber < target.maxPeriods
+                target.periodNumber < target.maxPeriods &&
+                target.isActive == true
             ) {
 
-                SafeERC20.safeApprove(token, gaugeList[idx], target.amountPerPeriod);
+                SafeERC20.safeApprove(token, ready[idx], target.amountPerPeriod);
 
                 try gauge.deposit_reward_token(address(token), uint256(target.amountPerPeriod)) {
                     s_targets[ready[idx]].lastInjectionTimeStamp = uint56(block.timestamp);
                     s_targets[ready[idx]].periodNumber += 1;
-                    emit emissionsInjection(ready[idx], address(token), target.amountPerPeriod);
+                    emit EmissionsInjection(ready[idx], address(token), target.amountPerPeriod);
                 } catch {
-                    emit injectionFailed(ready[idx]);
                     revert("Failed to call deposit_reward_tokens");
                 }
             }
         }
     }
 
+    /**
+ * * @notice This is to allow the owner to manually trigger an injection of funds in place of the keeper
+   * @notice without abi encoding the gauge list
+   * @param gauges array of gauges to inject tokens to
+   */
+    function injectFunds(address[] memory gauges) external onlyOwner {
+        _injectFunds(gauges);
+    }
+
+
     /**
      * @notice Get list of addresses that are ready for new token injections and return keeper-compatible payload
    * @param performData required by the chainlink interface but not used in this case.
```

Comment:
I'm assuming this is a typo? The (omitted) argument should not have the same as the return variable.